care/facility/api/serializers/facility.py

```diff
@@ -52,12 +50,10 @@ class FacilityBasicInfoSerializer(serializers.ModelSerializer):
     bed_count = serializers.SerializerMethodField()
 
     def get_bed_count(self, facility):
-        return Bed.objects.filter(facility=facility).count()
+        return facility.bed_set.count()
```

Comment:
would there be an issue here since bed_set might include deleted beds too (since deletes are soft-deletes in care)?

Reply:
cc: @sainak

Reply:
@rithviknishad the sql query made checks for not deleted 
<img width="1725" alt="Screenshot 2024-04-16 at 6 24 21 PM" src="https://github.com/coronasafe/care/assets/85862184/0881c7f6-28e8-4849-9833-4806c79646f2">